src/standalone/Soil/Biogeochemistry/Biogeochemistry.jl

```diff
@@ -306,6 +307,8 @@ struct SoilDrivers
     met::AbstractSoilDriver
     "Soil SOM driver - Prescribed only"
     soc::AbstractSoilDriver
+    "Prescribed atmospheric pressure"
+    atm_p::PrescribedAtmosphere
```

Comment:
@kmdeck I imagine this is where we would have atmospheric pressure (I will remove it from parameters), and I think the type is `::PrescibedAtmosphere` since this exist already, however it now feels weird to put this in `AbstractSoilDriver`. 

I am not sure what the most elegant way to do this is. 
I imagine we could rename `SoilDriver` to `Driver`, or have a struct that hold `SoilDriver` and `AtmosphericDriver`, ...?

Reply:
yeah! that makes sense to me. and then the SoilDriver can be prescribed or prognostic, and eventually the atmos driver can be prescribed or coupled?

FWIW, I believe both the canopy and the soil model have copies of the prescribed Atmos struct in their models, so having it here too is probably fine. Or, if you want, you can just store the co2 and pressure, and can extract them from the PrescribedAtmos object in the constructor for the integrated soil_canopy model?

[For global runs, we'll have the atmos variables in `p`, and then any model can access them that way]